src/scripts/utils/AppUtils.js

```diff
@@ -46,3 +46,15 @@ export const getParatiiConfig = (env: ?string = 'development'): Object => {
 
   return config
 }
+
+export const prettyBytes = (num: ?number): string => {
+  if (num === null || num === undefined) {
+    num = 0
+  }
+  const UNITS = ['B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
```

Comment:
Can these be moved to `ApplicationConstants.js` and imported into `AppUtils.js`? It is possible we may want to use this list elsewhere. Not a big deal though, can always move later.